src/controllers/pods.ts

```diff
@@ -153,6 +153,10 @@ export default function (server: Hapi.Server, deps: Injector) {
         `manifestHash=${manifestHash}`)
     }
 
+    if (manifest.unlisted) {
+      throw Boom.forbidden('This manifest is unlisted.')
```

Comment:
This defeats some of the purpose of having it unlisted, because it gives away the fact that it's running. Instead it should give an identical error to the case where it's not running at all.

Reply:
Seems it would be the same error as `throw Boom.serverUnavailable('pod has stopped. ' + `manifestHash=${manifestHash}`)`, should we consider changing that error as well. 

Oh its a 500 so the return to the user does not include the message, it will just get logged so just do a Boom.serverUnavailable('the manifest is unlisted') so we get the logs but the user sees the same thing.

Reply:
Yeah, [Boom documentation](https://github.com/hapijs/boom) states that for all 5XX errors it suppresses the messages from the response. It would probably be fine to log it for server admins (since they'd already know about the pods via Admin API).